### PR TITLE
pushing back some of the work we've done for zonar ci.

### PR DESCRIPTION
**Pull request by maratoid**
I'm not claiming this is functional for k2 out of the box, but we need a place other than zonar charts to keep charts for public-facing jenkins (and its components)

### CODE REVIEW

File: central-logging/requirements.lock [hunk 1]
```diff
@@ -0,0 +1,18 @@
+dependencies:
```
Comment (coffeepac) [inline central-logging/requirements.lock]: This is all 0.2.0 now.  Converted to StatefulSets

File: jenkins-base/templates/cluster-secrets-cleanup.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    spec:
+     restartPolicy: "Never"
+     containers:
+      - name: {{ template "hookname" . }}
+        image: google/cloud-sdk 
+        command: 
+          - "/bin/bash"
+          - "-cx"
+          - |
+            # delete pre install hook secrets
```
Comment (coffeepac) [inline jenkins-base/templates/cluster-secrets-cleanup.yaml]: When would these fail?  And why do we swallow the error?

File: k2-jenkins/README.md [hunk 3]
```diff
@@ -1,7 +0,0 @@
-# kraken-ci-chart
```
Comment (coffeepac) [inline k2-jenkins/README.md]: There should be some kind of light readme

**Comment by coffeepac** [comment]
In general, I don't know what this is supposed to be doing.  It also seems to be more then just CI work as it has the dns hackery and a chart for rancher which I am incredibly curious about.

I've heard there is an arch diagram for zonar's CI system.  I would love to see that and get a walkthrough if there's time.  I think after Wednesday maybe?

File: k2-jenkins/README.md (hunk 3 above)
Comment (l337ch) [inline k2-jenkins/README.md]: We will have readme(s) added in our next sprint

**Comment by l337ch** [comment]
There are documents in Confluence. I linked them to you in Slack. Where would you like these posted so that they are available publicly, Github wikis?

File: k2-jenkins/README.md (hunk 3 above)
Comment (coffeepac) [inline k2-jenkins/README.md]: thanks.

**Comment by coffeepac** [comment]
@l337ch I saw the linked document in slack, thank you.  We can sort out where to publish them later.  probably a github wiki but it'll be a convo with @iredwards

**Comment by coffeepac** [comment]
I'm going to merge this and I'll have a follow-PR shortly to clean up the lockfile messes that are being made here.

**Pull request merged**